architecture/classes.puml

```diff
@@ -37,7 +37,7 @@ class User {
 
 Place ---> Memory : contains
 Memory ---> Place : is associated to
```

Comment:
This should also be an aggregation, because otherwise it would lead to a memory leak.